base/server/src/main/java/com/netscape/cmscore/usrgrp/UGSubsystem.java

```diff
@@ -698,6 +698,14 @@ public void addUser(User user) throws EUsrGrpException {
             attrs.add(attr);
         }
 
+        List<LDAPAttribute> attributes = user.getAttributes();
+        if (attributes != null && attributes.size() > 0) {
+            for (LDAPAttribute attr : attributes) {
+                logger.debug("UGSubsystem: - " +  attr.getName() + ": " + attr.getStringValueArray());
+                attrs.add(new LDAPAttribute(attr));
+            }
+        }
+
```

Comment:
The `attributes.size() > 0` check might be redundant since the `for` loop will not do anything if `attributes` is empty.

Reply:
Ack, I cargo-culted the bit for the TPS profiles. I have changed this anyway in next revision to use `isEmpty()` so I might as well just drop it.